commit 3e0599ae0391733c19fd2253a27ab932a3d20bd9 (refs/changes/41/741/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-19T15:09:53-07:00 (3 years ago)
    
    MANTA-3009 Update muskie node version for CVE-2016-5180
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/Makefile b/Makefile
index 6fce20d..64b5393 100644
--- a/Makefile
+++ b/Makefile
@@ -55,7 +55,7 @@ CLEAN_FILES += node_modules
 #
 NAME 			= muskie
 NODE_PREBUILT_TAG       = zone
-NODE_PREBUILT_VERSION	:= v0.10.47
+NODE_PREBUILT_VERSION	:= v0.10.48
 NODE_PREBUILT_IMAGE     = fd2cc906-8938-11e3-beab-4359c665ac99
 
 include ./tools/mk/Makefile.defs
